src/session_test.cpp

```diff
@@ -54,7 +54,7 @@ DAP_STRUCT_TYPEINFO(TestResponse,
 struct TestRequest : public Request {
   using Response = TestResponse;
 
-  boolean b;
+  boolean req_b;
```

Comment:
What was the purpose of this rename? Looks like `b` has no collisions.

Reply:
I rename field b. to make it has a filed that hase the diffrent name with the field in TestRequest.
So i test it with the preversion code, I found the test case run faild.Rename it ,to make the typeinfo is diffrent betweend TestRequest and TestResponse.

Reply:
Okay, makes sense. I'll follow up with another change to rename all the fields with a `req_` and `res_` prefix for consistency.